### PR TITLE
Update Docs - webOS Required Port for segmented Networks

### PR DESCRIPTION
**Pull request by ryoku-cha**
Verified with a LG OLED C9 TV in a segmented network.
I always appreciate knowing which Ports need to be opened. Hopefully others will to.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/webostv.markdown [hunk 1]
```diff
@@ -239,3 +239,7 @@ automation:
         data:
           icon: "/home/homeassistant/images/doorbell.png"
 ```
+
+## Notes
+
+- If Home Assistant and your TV are not on the same subnet, you need to open Port 3000 with Protocol TCP from Home Assistant to your TV.
```
Comment (frenck) [inline source/_integrations/webostv.markdown]: I'm confused about how subnets are related to the port? Don't you mean the firewall or more explicit routing you might have placed between your subnets?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/webostv.markdown (hunk 1 above)
Comment (ryoku-cha) [reply source/_integrations/webostv.markdown]:
```suggestion
- If Home Assistant and your TV are not on the same network, you need to create a Firewall Rule, which allows a Connection on Port 3000 with Protocol TCP from Home Assistant to your TV.
```
I tried to keep this as simple as possible, but I agree that the wording was not ideal.
Hopefully my suggestion makes it more clear.

**Comment by ryoku-cha** [comment]
@frenck thanks for pointing out the unclear wording.
-> I just updated the wording. Hopefully this clarifies it.

(I'm still very new to GitHub and it's Workflows, so i hope i did this correctly)

File: source/_integrations/webostv.markdown [hunk 2]
```diff
@@ -239,3 +239,7 @@ automation:
         data:
           icon: "/home/homeassistant/images/doorbell.png"
 ```
+
+## Notes
+
+- If Home Assistant and your TV are not on the same network, you need to create a Firewall Rule, which allows a Connection on Port 3000 with Protocol TCP from Home Assistant to your TV.
```
Comment (frenck) [inline source/_integrations/webostv.markdown]:
```suggestion
If Home Assistant and your TV are not on the same network, you need to create a firewall rule, which allows a connection on port 3000 with the TCP protocol from Home Assistant to your TV.
```

**Review by frenck — APPROVED** [review]
Made a small tweak. Other than that 👍 

Thanks @ryoku-cha 👍

**Pull request merged**